example/90-shoot.yaml

```
@@ -339,6 +339,8 @@ spec:
   #   ignoreDaemonsetsUtilization: false
   #   verbosity: 2
   # verticalPodAutoscaler:
+  #   featureGates:
+  #     SomeKubernetesFeature: true
```

We can use an actual feature name here instead of a placeholder

```suggestion
  #     InPlaceOrRecreate: true
```

---

Thanks for pointing it, I had the same thought while adding it, but decided to leave it with `SomeKubernetesFeature` for consistency with the rest of the components. How do you find switching it to `SomeVpaFeature` as it's a _vpa_ related one ? I am intentionally avoiding the actual feature gate, although it might be unnecessary in the _vpa_ case ( as it's the only one as of now ), to keep the example spec neutral.

---

Let's keep `SomeKubernetesFeature: true`. We avoid putting real feature gates in the example docs as soon or later they get outdated because the feature gate is removed.